iotdb-cli/src/main/java/org/apache/iotdb/cli/client/AbstractClient.java

```diff
@@ -424,7 +422,7 @@ protected static void printName(boolean printTimestamp, int colCount,
         System.out.printf(formatValue, resultSetMetaData.getColumnLabel(i));
       }
     }
-    System.out.printf("\n");
+    System.out.printf("%n");
```

Comment:
why replace with '%'?

Reply:
'\n' does not mean a line separator in all OSs, while Java interpret '%n' as a proper line separator.

Reply:
For this case, we can use `System.out.println()`...